official/AnalisiDeiRequisiti/res/sections/editor.tex

```diff
@@ -593,7 +610,8 @@ \subsubsection{Aggiungere un riferimento ad un Column Element}
         
         \textbf{Precondizioni}  & Il \glossaryItem{Document Element} esiste nella sessione corrente dell'editor. \\ 
         
-        \textbf{Postcondizioni} & Il sistema aggiunge un riferimento tra il \glossaryItem{Column Element} e il \glossaryItem{Document Element}.
+        \textbf{Postcondizioni} & Il sistema aggiunge un riferimento tra il \glossaryItem{Column Element} e il \glossaryItem{Document Element}.\\
+        \textbf{Scenario principale} & 1. L'utente aggiunge un'\glossaryItem{associazione per riferimento} tra il \glossaryItem{Column Element} e l'attributo \glossaryItem{Populate} del \glossaryItem{Document Element}. \\ 
```

Comment:
@mdipirro non puoi dire a @drigoni di fare una pagina in php su tracy che mi faccia scaricare il file xml? Così non dobbiamo tenere l'xml sul repository ma lo scarico a ogni build con tracy/jenkins (e ogni volta è aggiornata).

Sennò ritorniamo a scrivere il glossario a mano solo che lo scriviamo in xml.